### PR TITLE
TSK-1317: Fixed task comment authorization tests.

### PR DESCRIPTION
**Pull request by holgerhagen**
https://sonarcloud.io/dashboard?branch=tsk1317&id=holgerhagen_taskana

### CODE REVIEW

File: rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java [hunk 1]
```diff
@@ -30,9 +30,8 @@
   public static final String URL_VERSION = PRE + "version";
   public static final String URL_TASKS = PRE + "tasks";
   public static final String URL_TASKS_ID = URL_TASKS + "/{taskId}";
-  public static final String URL_TASK_GET_POST_COMMENTS = URL_TASKS_ID + "/comments";
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java:33]: Is the removal of this url breaking?

File: rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java [hunk 2]
```diff
@@ -30,9 +30,8 @@
...
-  public static final String URL_TASK_COMMENTS = URL_TASKS + "/comments";
-  public static final String URL_TASK_COMMENT = URL_TASK_COMMENTS + "/{taskCommentId}";
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java:35]: The change of these Url's are breaking aswell, right?

File: rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java (hunk 1 above)
Comment (holgerhagen) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java:33]: I'm not removing anything. I'm just refactoring...

File: rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java (hunk 2 above)
Comment (holgerhagen) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java:35]: no. See above.

Comment (holgerhagen) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java:35]: If I would have changed anything, this should have an impact on the Controller, shouldn't it?

**Pull request merged**